src/styles/sidebar/components/markdown-editor.scss

```diff
@@ -0,0 +1,44 @@
+$toolbar-border: 0.1em solid $grey-3;
+
+.markdown-editor__toolbar {
+  display: flex;
+  flex-direction: row;
+
+  background-color: $white;
+  border-top: $toolbar-border;
+  border-left: $toolbar-border;
+  border-right: $toolbar-border;
+  border-radius: 0.15em 0.15em 0 0;
+  width: 100%;
+  margin-bottom: -0.1em;
+  padding: 5px 5px;
+  user-select: none;
```

Comment:
Can you enlighten me on the purpose of this rule? I'm new to `user-select`; looked it up (and noted lack of support in IE) but perhaps a comment here could help?

Reply:
It turns out that `user-select` is now obsolete. I've removed it.